packages/toolpad-app/src/toolpad/AppEditor/PagesExplorer/index.tsx

```diff
@@ -365,7 +365,7 @@ export default function PagesExplorer({ className }: PagesExplorerProps) {
             key={page.id}
             nodeId={page.id}
             toolpadNodeId={page.id}
-            labelText={page.name}
+            labelText={page.attributes.displayName ?? page.name}
```

Comment:
```suggestion
            labelText={page.name}
```

We just need to change this back and then I think it's good to go!